app/services/pages/page.rb

```diff
@@ -0,0 +1,15 @@
+module Pages
+  class Page < ViewItem
+    attr_reader :url_hash, :view
+
+    def initialize(url_hash, view)
+      @url_hash = url_hash
+      @view = view
+      super()
+    end
+
+    def name
+      "#{@url_hash[:controller]}/#{@url_hash[:action]}".to_sym
+    end
+  end
+end
```

Comment:
This class is no longer necessary and you can also get rid of inheritance completely.